arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+        }
+      },
+      "description": "Contains the information for blobs. Each one is used for a specific time range."
+    },
+    "BlobLocation": {
+      "properties": {
+        "blobEndpoint": {
+          "type": "string",
+          "description": "Gets or sets the blob endpoint."
+        },
+        "blobInfo": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BlobInfo"
+          },
+          "description": "Gets or sets the blob info."
+        }
+      },
+      "description": "Details about the set of blobs containing data split by time."
+    },
+    "LogDefinition": {
+      "properties": {
+        "category": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "Gets or sets the category of the log."
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "Gets or sets the resource identifier of the resource that has emitted the log."
+        },
+        "retention": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the retention."
+        },
+        "blobLocation": {
+          "$ref": "#/definitions/BlobLocation",
+          "description": "Gets or sets the location of the blob."
+        }
+      },
+      "description": "Log definition class specifies the metadata for a log."
+    },
+    "LogDefinitionCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LogDefinition"
+          },
+          "description": "Gets or sets the values for the log definitions."
+        }
+      },
+      "description": "Represents collection of log definitions."
+    },
+    "LogValue": {
+      "properties": {
+        "time": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the time when the log was generated"
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "Gets or sets id of the reosurce related to the log."
+        },
+        "operationName": {
+          "type": "string",
+          "description": "Gets or sets the name of the operation."
+        },
+        "operationVersion": {
+          "type": "string",
+          "description": "Gets or sets the version of the operation."
+        },
+        "category": {
+          "type": "string",
+          "description": "Gets or sets the category of the log."
+        },
+        "resultType": {
+          "type": "string",
+          "description": "Gets or sets the status of the operation."
+        },
+        "resultSignature": {
+          "type": "string",
+          "description": "Gets or sets the substatus of the operation."
+        },
+        "resultDescription": {
+          "type": "string",
+          "description": "Gets or sets the substatus of the operation."
```

Comment:
description for resultType, resultDescription, resultSignature are inaccurate. We have an ongoing project to generate MSDN documentation of the service from the swagger specs. Hence it is very important to document things properly.